src/internal/components/chart-plot/index.tsx

```diff
@@ -218,7 +218,11 @@ function ChartPlot(
         </g>
       </svg>
 
-      <LiveRegion>{ariaLiveRegion}</LiveRegion>
+      {typeof ariaLiveRegion === 'string' ? (
+        <LiveRegion>{ariaLiveRegion}</LiveRegion>
+      ) : (
+        <LiveRegion source={[ariaLiveRegion]} />
```

Comment:
Use the new `source` attribute introduced in #1734 for content with potentially navigable elements